CMakeLists.txt

```diff
@@ -2,10 +2,10 @@ cmake_minimum_required(VERSION 3.5)
 project(wirehair)
 
 set(CMAKE_CXX_STANDARD 11)
-find_program(CXX_10 g++-10)
-if (CXX_10)
-  set(CMAKE_CXX_COMPILER ${CXX_10})
-endif()
+# find_program(CXX_10 g++-10)
```

Comment:
Do you want to comment out this section? If so, let's remove the comment.

Reply:
Removed!